api/metadata_handler.go

```diff
@@ -121,13 +126,27 @@ func handleMetadataTriage(ctx context.Context, gac shared.GitHubAccessControl, t
 	}
 
 	// TODO(kyleju): Check github client permission levels for auto merge.
-	pr, err := tm.Triage(metadata)
+	prURL, err := tm.Triage(metadata)
 	if err != nil {
 		http.Error(w, "Unable to triage metadata: "+err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	w.Write([]byte(pr))
+	prArray := strings.Split(prURL, "/")
+	prNum := prArray[len(prArray)-1]
```

Comment:
Nit: to code defensively, we may want to verify that `prArray` has non-zero length, and `http.Error` if it doesn't.